---
# Documentation: https://wowchemy.com/docs/managing-content/

title: "Building Random Forest QSAR Models for Affinity Identification of 14-3-3 ζ with Optimized Parameters"
authors: [Ying Fan, Xiaojun Wang, Chao Wang]
date: 2020-10-18
doi: "10.1145/3431943.3431951"

# Schedule page publish date (NOT publication's date).
publishDate: 2020-12-01T15:26:53+08:00

# Publication type.
# Legend: 0 = Uncategorized; 1 = Conference paper; 2 = Journal article;
# 3 = Preprint / Working Paper; 4 = Report; 5 = Book; 6 = Book section;
# 7 = Thesis; 8 = Patent
publication_types: ["1"]

# Publication name and optional abbreviated publication name.
publication: "*Association for Computing Machinery*"
publication_short: "ACM"

abstract: "*Aspergillus oryzae*, an important filamentous fungus used in food fermentation and the enzyme industry, has been shown through genome sequencing and various other tools to have prominent features in its genomic composition. However, the functional complexity of the A. oryzae transcriptome has not yet been fully elucidated. Here, we applied direct high-throughput paired-end RNA-sequencing (RNA-Seq) to the transcriptome of A. oryzae under four different culture conditions. With the high resolution and sensitivity afforded by RNA-Seq, we were able to identify a substantial number of novel transcripts, new exons, untranslated regions, alternative upstream initiation codons and upstream open reading frames, which provide remarkable insight into the A. oryzae transcriptome. We were also able to assess the alternative mRNA isoforms in A. oryzae and found a large number of genes undergoing alternative splicing. Many genes and pathways that might be involved in higher levels of protein production in solid-state culture than in liquid culture were identified by comparing gene expression levels between different cultures. Our analysis indicated that the transcriptome of A. oryzae is much more complex than previously anticipated, and these results may provide a blueprint for further study of the A. oryzae transcriptome."
abstract: "14-3-3s present in multiple isoforms in human cells and mediate signal transduction by binding to phosphoserine-containing proteins. Previous studies demonstrate that the isoform 14-3-3 ζ acts as a key factor in promoting chemoresistance of cancer. Here, our work is devoted to developing the predictive models that can determine the binding affinity of phosphopeptide fragments against 14-3-3 ζ by the random forest approach. Based on the variable matrix built by the simple descriptors DPPS and statistical methods coupled with optimized hyperparameters, the robust models are obtained by a combinatorial peptide microarray dataset (n = 385 for N-terminal sublibrary, n = 384 for C-terminal sublibrary). For the test set, the R2 and RMSE are 0.8532 and 0.4516 at the N-terminal sublibrary (n = 96) and are 0.7998 and 0.5929 at the C-terminal sublibrary (n = 94), respectively. We also find that the distinct physiochemical properties function on the 14-3-3 ζ interaction. Overall, our results demonstrate that the computational methods based on QSAR analysis can be used for building the predictive models on the binding affinity of phosphopeptide against 14-3-3 ζ and contribute to the further research on clinical research."

# Summary. An optional shortened abstract.
summary: ""

tags: []
categories: []
featured: false

# Custom links (optional).
#   Uncomment and edit lines below to show custom links.
# links:
# - name: Follow
#   url: https://twitter.com
#   icon_pack: fab
#   icon: twitter

url_pdf:
url_code:
url_dataset:
url_poster:
url_project:
url_slides:
url_source:
url_video:

# Featured image
# To use, add an image named `featured.jpg/png` to your page's folder. 
# Focal points: Smart, Center, TopLeft, Top, TopRight, Left, Right, BottomLeft, Bottom, BottomRight.
image:
  caption: ""
  focal_point: ""
  preview_only: false

# Associated Projects (optional).
#   Associate this publication with one or more of your projects.
#   Simply enter your project's folder or file name without extension.
#   E.g. `internal-project` references `content/project/internal-project/index.md`.
#   Otherwise, set `projects: []`.
projects: [content/project/microarray1433/index.md]

# Slides (optional).
#   Associate this publication with Markdown slides.
#   Simply enter your slide deck's filename without extension.
#   E.g. `slides: "example"` references `content/slides/example/index.md`.
#   Otherwise, set `slides: ""`.
slides: ""
---
